fixed-hash/src/lib.rs

```diff
@@ -8,6 +8,11 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+// Re-export libcore using an alias so that the macros can work without
+// requiring `extern crate core` downstream.
+#[doc(hidden)]
+pub extern crate core as core_;
```

Comment:
Hmm. This is somewhat unsatisfying. Isn't it normal for a `no_std` crate to add `core` (if they need it). With this a downstream crate will add `core` implicitly which is perhaps undesired? 
What is the alternative to this, if any?

Reply:
As far as @pepyakin and me found out this is the general workaround for the problem.
Also used in other crates, such as `static_assertions`.